src/main/java/org/jitsi/protocol/xmpp/JingleUtils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.protocol.xmpp;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jingle.*;
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jitsimeet.*;
+import org.jitsi.impl.protocol.xmpp.extensions.*;
+
+/**
+ * Jicofo specific utilities for Jingle.
+ *
+ * @author Boris Grozev
+ */
+public class JingleUtils
+{
+    /**
+     * Adds a group packet extension to a {@link JingleIQ}, and a
+     * {@link BundlePacketExtension} to each of its contents. I.e. adds
+     * everything that we deem necessary to turn enable {@code bundle} in an
```

Comment:
extra word `turn`?  or maybe i'm misreading this (as I'm not sure I quite follow what it's saying)